src/api/app/views/webui/obs_factory/distributions/show.html.haml

```diff
@@ -78,7 +78,7 @@
       - if @versions[:totest]
         %li
           Testing:
-          = link_to @versions[:totest], "#{openqa_links_helper}/tests/overview?distri=opensuse&version=#{@distribution.openqa_version}&group=#{@distribution.openqa_group}"
+          = link_to @versions[:totest], "#{openqa_links_helper}/tests/overview?distri=opensuse&version=#{@distribution.openqa_version}"
```

Comment:
I think you will not get a result as you expected, this matches so many, some more condition is required to get a squashed result.

Reply:
I was wondering what this link should show anyway, should we try to find out the most recent build or filter by flavors?

Reply:
I think we can even drop the distri=opensuse - would make it a little more useful for others :)